sys/ztimer/Makefile.dep

```diff
@@ -74,4 +74,8 @@ endif
 
 ifneq (,$(filter ztimer_msec,$(USEMODULE)))
   USEMODULE += ztimer
+  FEATURES_OPTIONAL += periph_rtt
+  ifneq (,$(filter periph_rtt,$(FEATURES_USED)))
+    USEMODULE += ztimer_periph_rtt
+  endif
```

Comment:
I like this approach! It's heading more in the direction of *it simply works*.

Reply:
I think I'll split the first two commits out of this one!